tw-tasks-core-spring-boot-starter/src/main/java/com/transferwise/tasks/core/autoconfigure/TwTasksCoreAutoConfiguration.java

```diff
@@ -72,10 +71,6 @@
 @AutoConfigureAfter({ValidationAutoConfiguration.class})
 public class TwTasksCoreAutoConfiguration {
 
-  // Following is not used by the code, but makes sure, that someone has not turned graceful shutdown completely off.
```

Comment:
This was necessary in earlier days, when we did not add tw-graceful-shutdown as runtime dependency.